org.scala-ide.sdt.core/plugin.xml

```diff
@@ -1838,6 +1838,42 @@
           The font used for all Scala hovers.
        </description>
     </fontDefinition>
+    <colorDefinition
+          categoryId="org.scalaide.ui.theme"
+          id="org.scalaide.ui.color.interpreterForeground"
+          label="Interpreter foreground color"
+          value="60,0,128">
```

Comment:
The difference is that `defaultsTo` falls back to a color ID, like the background color of the text editors. That means there's an additional level of indirection. I'm pretty sure ColorThemes doesn't change all colors, but just the ones that come from the platform and a few Java ones. We could patch ColorThemes, but if it works without, it's simpler.